dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -3031,11 +3032,32 @@ public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop
      * @param  skipArchives           Skip postStopServer collection of archives (WARs, EARs, JARs, etc.) - only used if postStopServerArchive is true
      * @param  ignoredFailuresRegExps A list of reg expressions corresponding to warnings or errors that should be ignored.
      *                                    If ignoredFailuresRegExps is null, logs will not be checked for warnings/errors
+     * @param  errorRegexes           A list of reg expressions corresponding to warnings or errors that should be treated as test failures.
      * @return                        the output of the stop command
      * @throws Exception              if the stop operation fails or there are warnings/errors found in server
      *                                    logs that were not in the list of ignored warnings/errors.
      */
     public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop, boolean skipArchives, String... ignoredFailuresRegExps) throws Exception {
+        List<String> failuresRegExps = Arrays.asList(LIBERTY_ERROR_REGEX);
+        return stopServer(postStopServerArchive, forceStop, true, failuresRegExps, ignoredFailuresRegExps);
+    }
```

Comment:
The `errorRegexes` parameter should be removed from the javadoc since this method doesn't have it

Reply:
Oops. I renamed the new paramater to failuresRegExps to match the old paramater and forgot to update the name in the javadoc.